app-admin/filebeat/filebeat-5.6.8.ebuild

```diff
@@ -0,0 +1,69 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Lightweight log shipper for Logstash and Elasticsearch"
+HOMEPAGE="https://www.elastic.co/products/beats"
+SRC_URI="https://github.com/elastic/beats/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=dev-lang/go-1.9.2"
+RDEPEND="!app-admin/filebeat-bin"
+
+ELASTIC="${WORKDIR}/src/github.com/elastic"
+BEATS="${ELASTIC}/beats"
+S="${BEATS}"
+
+src_unpack() {
+	mkdir -p "${ELASTIC}" || die
+	unpack ${P}.tar.gz
+	mv beats-${PV} "${BEATS}" || die
+}
+
+src_compile() {
+	cd ${BEATS}/filebeat || die
+	GOPATH="${WORKDIR}" emake
```

Comment:
`emake -C ...` instead of `cd`. Also, please don't use absolute path when relative path is much simpler.